.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -3,7 +3,7 @@
 blank_issues_enabled: true
 contact_links:
 - name: Ask a question
-  url: https://github.com/k8s-at-home/charts/discussions
+  url: https://github.com/k8s-at-home/library-charts/discussions
```

Comment:
What do you think about only having discussions enabled on that repo? sucks we can't have a org level discussion board.